.github/workflows/conformance.yml

```diff
@@ -61,6 +59,7 @@ jobs:
           ./hack/build-antrea-linux-all.sh --pull
       - name: Install Kind
         run: |
+          KIND_VERSION=$(head -n1 ./ci/kind/version | xargs)
```

Comment:
AFAIK, using `xargs` by itself like this is for converting multi-line output into single line
It doesn't seem useful here, as we will always have a single line in `./ci/kind/version`

Reply:
Actually i using `xargs` to remove extra whitespace from the kind version, because it was showing the URL format error in curl cmd because of extra space after kind version.

KIND_VERSION="$(head -n1 ./ci/kind/version)"
https://github.com/kubernetes-sigs/kind/releases/download/v0.20.0  /kind-Linux-amd64
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100     9  100     9    0     0     60      0 --:--:-- --:--:-- --:--:--    61
curl: (3) URL using bad/illegal format or missing URL

should I use some other cmd ?

Reply:
that's because you have extra whitespaces at the end of the first line in `ci/kind/version`, which doesn't need to be here.
the file contents are `v0.20.0  ` instead of `v0.20.0`

Reply:
my bad.., I didn't notice that i removed it, thank you!

Reply:
you still use `xargs` in a few places

Reply:
done